frr-tools/metrics/collector/bfd_test.go

```diff
@@ -197,11 +201,19 @@ func TestBFDCollect(t *testing.T) {
 
 			l := log.NewNopLogger()
 			collector := NewBFD(l)
+			cmdOutput := map[string]string{
+				"show bgp vrf all json":                    vrfVtysh,
+				"show bfd vrf default peers json":          test.vtyshPeersOutput,
+				"show bfd vrf red peers json":              "[]",
```

Comment:
wdyt about adding a test that has peers in another vrf?

Reply:
Adding a test with a different vrf won't change how the metrics look like (because "default" is a value like the others). We could add a test with two peers, but would overcomplicate the test and we still have e2e to cover that.